src/components/MessageLogger/faultsAndWarningList.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from "react";
+import { ConsoleList } from "@components/MessageLogger/ConsoleList";
+import { Message } from "@components/MessageLogger/structs/Message";
+
+
+export const FaultsAndWarningList = () => {
+    var warningMessages: Message[] = [{ id: "1", desc: "Warning: Each child in a list should have a unique key prop" }, 
```

Comment:
prohibido hacer esto, hay maneras mucho mejores. estos mensajes deberian estar definidos o bien en back, o bien en un txt/csv/dotenv/properties que funcione a modo de "base de datos" (salvando las distancias)

Reply:
no se si esta hecho para probar, pero aun asi, no le debe corresponder a la capa de presentación gestionar la logica de crear mensajes, ni siquiera saber que es un mensaje, esto tiene que ir fuera

Reply:
It is simply to represent it on the front end to see how it would look like, those messages are received from the back end, they are not created here. We have to implement the passage of these from the back to the front.